operator/api/v1alpha1/virtualippool_types.go

```diff
@@ -22,7 +22,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-const configuredConditionName = "Configured"
+const (
+	configuredConditionName = "Configured"
+	availableConditionName  = "Available"
+	readyConditionName      = "Ready"
```

Comment:
Not sure if that's the right place, or if we should add it to the `description` in OpenAPI spec, but it would be nice to explain each condition type and their nuances (e.g. Available = at least one up, Ready = all up).

Reply:
Ok I will add some explanation

Reply:
Done https://github.com/scality/metalk8s/pull/3864/commits/d79b19c49a776c53abf77dcc3e68fa858351a664